source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -2499,8 +2487,8 @@ AdaptiveNamespaceStart
         [propget] HRESULT IconUrl([out, retval] HSTRING* value);
         [propput] HRESULT IconUrl([in] HSTRING value);
 
-        [propget] HRESULT Sentiment([out, retval] Sentiment* value);
```

Comment:
I've added some UWP unit tests in master that probably won't build after this change. Please merge or rebase master and make sure they're okay before checkin, thanks!

Reply:
I already took care of those :)

---
In reply to: [250389032](https://github.com/Microsoft/AdaptiveCards/pull/2243#discussion_r250389032) [](ancestors = 250389032)